src/models/Bitcoin.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+export type BitcoinTransactionTarget = {
+  address: string,
+  value: number,
+  isChange?: boolean,
+};
+
+export type BitcoinUtxo = {
+  address: string,
+  txid: string,
+  vout: number,
+  scriptPubKey: string,
+  amount: number,
+  satoshis: number,
```

Comment:
Just wondering, in Ether transaction model we also have `amount` with decimal places and even though Ether transactions are actually in wei we keep only `amount` key in the mode. So just thinking, Bitcoin has even less decimals than Ether, so how about keeping `amount` key and parsing as `satoshis` only when needed?

Reply:
You are correct, we don't need both, but this is used to represent what insight returns, we don't build these. Since they return both, I thought it would make sense to declare both here.

In other hand, one of the key things we do with these values is select which ones will be used for a transaction. That job is done by `coinselect`, which expect satoshis but in a `value` key, so if we remove the satoshis we'd have to calculate it again on all the cases.

What do you think?

Reply:
If it's return type then I think it's fine 👍